### ISSUE TITLE
Transfer learning tutorial should set .eval before visualizing on val

### ISSUE THREAD

**Issue by SsnL** [pytorch/tutorials#203]
The resnet18 used has BN layers which needs to be in eval mode to give good results. I'll submit a PR tomorrow.

http://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html#sphx-glr-beginner-transfer-learning-tutorial-py

**Comment by chsasank** [comment]
Thanks. PR will be nice! 

You need to just add model.eval() to the top of `visualize_model`
